src/load.jl

```diff
@@ -0,0 +1,24 @@
+module Load
+
+using Libdl: Libdl
+using MPIPreferences: MPIPreferences
+import ..libmpi
+
+function __init__()
+    @static if MPIPreferences.binary == "system" && (x = tryparse(Bool, get(ENV, "CI", "false")); something(x, false))
```

Comment:
Like in #590, I wanted to have a way to actually check that we open libmpi where we want.  I tried to keep this check as little invasive as possible with the `@static if`, could also replace `CI` with a more specific environment variable like `GITHUB_ACTIONS`

Reply:
I like it!